packages/alpha/src/types.ts

```diff
@@ -73,6 +73,7 @@ export interface Simulator {
 
 export interface SimulatorFilterQuery extends FilterQuery {
   filter?: object;
+  limit?: number;
 }
 
```

Comment:
```suggestion
export interface SimulatorFilterQuery extends FilterQuery {
  filter?: object;
}

```
filter query has cursor and limit already

Reply:
🤦 

fixed